packages/web/src/server/index.ts

```diff
@@ -33,7 +34,7 @@ const logPrefix = '[Vercel Web Analytics]';
 export async function track(
   eventName: string,
   properties?: Record<string, AllowedPropertyValues>,
-  context?: Context
+  options?: Options
```

Comment:
**praise:** thanks for renaming this parameter in something more aligned with the other existing methods